main.py

```diff
@@ -0,0 +1,105 @@
+from typedb.client import *
+
+def highestVaccines():
+
+    with TypeDB.core_client("localhost:1729") as client:
+        with client.session("vaccination_schema", SessionType.DATA) as session:
+            with session.transaction(TransactionType.READ) as read_transaction:
+
+                answer_iterator = read_transaction.query().match_group_aggregate('match $p isa person, has country $c; get $p, $c; group $c; count;')

```

Comment:
This query doesn't count the number of vaccinations, it counts the number of people who have had vaccinations - if a person can have multiple vaccinations, these are distinct concepts.

For example, if your dataset contained 10 people from England who've had 1 vaccine each, and 6 people from the US who've had 2 vaccines each then there have been more vaccinations in the US, but more people vaccinated in England.

I appreciate that your data generator doesn't allow for the generation of people with multiple vaccinations, but I think it's worth making the change, since querying directly for what you're actually looking for is generally a much better idea.

Reply:
Here, in the given dataset, I have considered only one vaccination for each person, because I haven't created any entity such as vaccination_certificate_no, which can store the certificate no. (or any unique information) for every dose administered.
I think that though we can add an entirely new entity relating to vaccination record for each instance,
but in the given scenario what I have taken, it won't cause ambiguity.

Reply:
Altered the query, both in main.py and queries.tql